wallet_transactions/transactions/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+from wallets.models import Wallet
+
+
+PAID = "PAID"
+FAILED = "FAILED"
+ZEROCOMMISION = 0.00
```

Comment:
please, add _ in such variables between the words.
it will be easier to read it

Reply:
I'm add _